src/app/containers/SocialEmbed/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useContext } from 'react';
+import path from 'ramda/src/path';
+import styled from 'styled-components';
+import {
+  AmpSocialEmbed,
+  CanonicalSocialEmbed,
+} from '@bbc/psammead-social-embed';
+import { GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import useToggle from '#hooks/useToggle';
+import socialEmbedBlockPropTypes from '#models/propTypes/socialEmbed';
+import createTranslations from './translations';
+
+const MAX_WIDTH = '31.25rem';
+
+const Wrapper = styled.div`
+  margin-right: auto;
+  margin-left: auto;
+  margin-bottom: ${GEL_SPACING_TRPL};
+  max-width: ${MAX_WIDTH};
+
+  .twitter-tweet,
+  .instagram-media {
+    margin-top: 0 !important;
+    margin-bottom: 0 !important;
+  }
+  .instagram-media {
+    min-width: auto !important;
+  }
+`;
```

Comment:
I guess these rules are intentionally coupled to the instagram and twitter embed implementation? What problem are they solving? Just could become a problem as they could change their css without warning causing a visual regression.

Reply:
- Yes, this is intentional.
- We need to apply these styles in this way to reset provider styles. They protect against Instagram and Twitter applying a min-width and top and bottom margins. We can't stop providers from changing their CSS, but we can prevent them from setting certain rules.

Reply:
I may've confused things by saying 'protect' - they are overriding additional margins and minimum-widths we don't want applied by Twitter and Instagram at this time. These rules can be thought of as more of a 'normalisation' between the different providers.